test/court-disputes.js

```diff
@@ -141,9 +141,33 @@ contract('Court: Disputes', ([ poor, rich, governor, juror1, juror2, juror3, arb
       const firstRoundId = 0
 
       beforeEach(async () => {
-        await assertLogs(this.court.createDispute(arbitrable, rulings, jurors, term), NEW_DISPUTE_EVENT)
+        await assertLogs(this.court.createDispute(arbitrable, rulings, jurors, term, { from: poor }), NEW_DISPUTE_EVENT)
       })
 
+      it('gets the correct dispute details', async () => {
+        const dispute = await this.court.getDispute(disputeId)
+
+        assert.strictEqual(dispute[0], arbitrable, 'COURT_INCORRECT_DISPUTE_SUBJECT')
```

Comment:
The assertion messages that you have chosen here are have the same format as the smart contract errors (which are meant to be identifiers), but for the assertions we tend to just use more descriptive messages.

```suggestion
        assert.strictEqual(dispute[0], arbitrable, 'incorrect dispute subject')
```

Reply:
That makes sense, I noticed a mix of assert message formats but I missed that association. Thanks!